portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro

```diff
@@ -189,6 +189,25 @@ definition {
 		return "${curl}";
 	}
 
+	macro _getStructuredContentFoldersWithDifferentParameters {
+		var portalURL = JSONCompany.getPortalURL();
+		var api = "asset-libraries/${assetLibraryId}/structured-content-folders";
+
+		if (isSet(parameter)) {
+			var api = StringUtil.add("${api}", "?${parameter}=${parameterValue}", "");
+		}
+
+		var curl = '''
```

Comment:
please format to get an alignment like this one:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/35199951/185058757-2ca5e8f5-1f33-4ee4-885d-2eea660e741e.png">

Reply:
I have updated it.